superset-frontend/jest.config.js

```diff
@@ -29,6 +29,7 @@ module.exports = {
     // mapping plugins of superset-ui to souce code
     '@superset-ui/(.*)$': '<rootDir>/node_modules/@superset-ui/$1/src',
   },
```

Comment:
a minor comment would be useful for explanation

Reply:
This is a default configuration when using es6 modules with jest.